package/MDAnalysis/analysis/density.py

```diff
@@ -180,11 +177,259 @@
+    Normal :class:`AtomGroup` instances represent a static selection of
+    atoms. If you want *dynamically changing selections* (such as "name OW and
+    around 4.0 (protein and not name H*)", i.e., the water oxygen atoms that
+    are within 4 Å of the protein heavy atoms) then create an
+    :class:`~MDAnalysis.core.groups.UpdatingAtomGroup` (see Examples).
+
+
+    Examples
+    --------
+    A common use case is to analyze the solvent density around a protein of
+    interest. The density is calculated with :class:`DensityAnalysis` in the
+    fixed coordinate system of the simulation unit cell. It is therefore
+    necessary to orient and fix the protein with respect to the box coordinate
+    system. In practice this means centering and superimposing the protein,
+    frame by frame, on a reference structure and translating and rotating all
+    other components of the simulation with the protein. In this way, the
+    solvent will appear in the reference frame of the protein.
+
+    An input trajectory must
+
+    1. have been centered on the protein of interest;
+    2. have all molecules made whole that have been broken across periodic
+       boundaries [#pbc]_;
+    3. have the solvent molecules remapped so that they are closest to the
+       solute (this is important when using triclinic unit cells such as
+       a dodecahedron or a truncated octahedron) [#pbc]_;
+    4. have a fixed frame of reference; for instance, by superimposing a
+       protein on a reference structure so that one can study the solvent
+       density around it [#fit]_.
+
+    To generate the density of water molecules around a protein (assuming that
+    the trajectory is already appropriately treated for periodic boundary
+    artifacts and is suitably superimposed to provide a fixed reference frame)
+    [#testraj]_, first  create the :class:`DensityAnalysis` object by
+    supplying an AtomGroup, then use  the :meth:`run` method::
+
+        from MDAnalysis.analysis import density
+        u = Universe(TPR, XTC)
+        ow = u.select_atoms("name OW")
+        D = density.DensityAnalysis(ow, delta=1.0)
+        D.run()
+        D.convert_density('TIP4P')
+
+    The positions of all water oxygens are histogrammed on a grid with spacing
+    *delta* = 1 Å. Initially the density is measured in inverse cubic
+    ångström. With the :meth:`Density.convert_density` method,
+    the units of measurement are changed. In the example we are now measuring
+    the density relative to the literature value of the TIP4P water model at
+    ambient conditions (see the values in :data:`MDAnalysis.units.water` for
+    details). In particular, the density is stored as a NumPy array in
+    :attr:`grid`, which can be processed in any manner. Results are available
+    through the :attr:`density` attribute, which has the :attr:`grid`
+    attribute that contains the histogrammed density data.
+    :attr:`DensityAnalysis.density` is a :class:`gridData.core.Grid` object.
```

Comment:
Here (or earlier, since you `convert_density` is a Density function) could you explicitly say that it's a Density object, which is a subclass of Grid? Density has some extra methods defined and you can point users to those docs to work out what to do with their results.

Reply:
Also, an `Attributes` section in the docs (even for the one thing) is more immediately discoverable.